src/snmp/Forwarder.cc

```diff
@@ -76,7 +76,7 @@ Snmp::Forwarder::handleException(const std::exception& e)
 void
 Snmp::Forwarder::sendError(int error)
 {
-    debugs(49, 3, HERE);
+    debugs(49, 3, "");
```

Comment:
Agreed, but this PR automatically removes HERE, and nothing else should be required.